### PR TITLE
write out prediction options

### PR DESCRIPTION
**Pull request by feich-ms**
fix issue here https://github.com/microsoft/BotFramework-Composer/issues/2545

### CODE REVIEW

**Comment by feich-ms** [comment]
added all the properties

File: packages/lu/src/parser/lubuild/recognizer.ts [hunk 1]
```diff
@@ -36,6 +37,14 @@ export class Recognizer {
     this.applicationId = `=settings.luis.${targetFileName.split('.').join('_')}`
     this.endpoint = '=settings.luis.endpoint'
     this.endpointKey = '=settings.luis.endpointKey'
+    this.predictionOptions = {
+      includeAllIntents: '=coalesce(settings.luis.preditionOptions.includeAllIntents, false)',
+      includeInstanceData: '=coalesce(settings.luis.preditionOptions.includeInstanceData, false)',
+      log: '=coalesce(settings.luis.preditionOptions.log, false)',
```
Comment (luhan2017) [inline packages/lu/src/parser/lubuild/recognizer.ts:43]: Let's update the default value:

log = true
preferExternalEntities = true
slot = 'production'

Comment (luhan2017) [reply packages/lu/src/parser/lubuild/recognizer.ts:43]: @vishwacsena, we should add the default values in sdk schema. There are default values in sdk code but not in schema.

Comment (vishwacsena) [reply packages/lu/src/parser/lubuild/recognizer.ts:43]: @chrimc62 what should we default these values to?

Comment (chrimc62) [reply packages/lu/src/parser/lubuild/recognizer.ts:43]: includeAllIntents = false
includeInstanceData = true (Without this, entity events would not work.  You could make a case for this being false by default, but then your docs would have to tell people to turn it on--I'd rather it were true and someone that wants to make payloads smaller could turn it off.)
log = true
preferExternalEntities = true

Comment (feich-ms) [reply packages/lu/src/parser/lubuild/recognizer.ts]: changed to above default values. Please verify.

**Comment by feich-ms** [comment]
Can anyone approve this PR if it looks good to you? @vishwacsena @luhan2017

**Pull request merged**